index.js

```diff
@@ -9939,42 +10155,16 @@ module.exports = {
             }
         },
         'p-applicant-incident-description': {
-            l10n: {
-                vars: {
-                    lng: 'en',
-                    context: {
-                        $data:
-                            '/answers/p-applicant-who-are-you-applying-for/q-applicant-who-are-you-applying-for'
-                    },
-                    ns: 'p-applicant-incident-description'
-                },
-                translations: [
-                    {
-                        language: 'en',
-                        namespace: 'p-applicant-incident-description',
-                        resources: {
-                            'q-applicant-incident-description': {
-                                error: {
-                                    required: 'Describe the crime in your own words',
-                                    'required_someone-else': 'Describe the crime'
-                                },
-                                title: 'Briefly describe the crime in your own words',
-                                'title_someone-else': 'Briefly describe the crime'
-                            }
-                        }
-                    }
-                ]
-            },
             schema: {
                 $schema: 'http://json-schema.org/draft-07/schema#',
                 type: 'object',
                 required: ['q-applicant-incident-description'],
                 properties: {
                     'q-applicant-incident-description': {
                         type: 'string',
-                        title: 'l10nt:q-applicant-incident-description.title{?lng,context,ns}',
```

Comment:
same as above. These were different in the prototype (for minors, compared to self journey). Has Julie etc. just asked that we leave these ones?

Reply:
That was an oversight, Julie said they should be consistent across both journeys, not different.